bundles/io/org.eclipse.smarthome.io.rest/src/main/java/org/eclipse/smarthome/io/rest/JSONResponse.java

```diff
@@ -142,11 +145,18 @@ public static Response createResponse(Response.Status status, Object entity, Str
      */
     @Provider
     public static class ExceptionMapper implements javax.ws.rs.ext.ExceptionMapper<Exception> {
+
+        private final static Logger logger = LoggerFactory.getLogger(ExceptionMapper.class);
```

Comment:
@kaikreuzer IIRC we tried to use static loggers only if we need them in static functions.